src/random/pcg.cr

```diff
@@ -0,0 +1,80 @@
+# This is a Crystal conversion of basic C PCG implementation
```

Comment:
The naming is because there also exists PCG64 and some variations with different seeding that share most of code, so i thought that they could be added to this file later. On the other hand, right now it doesn't make sense so i'll change it.